notebooks/2.1.1-he-rain-analysis.ipynb

```diff
@@ -72,22 +72,31 @@
    "metadata": {},
```

Comment:
<p>The second and third shouldnt be consensus, but just what the station closest in location/elevation reported for circles that reported rain</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataKind-DC/audubon-cbc/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='110'/>

Reply:
<p>Fixed in Rain, snow, and snowfall notebooks</p>